proknow-sdk-test/LogTest/AuditTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+using System.IO;
+using ProKnow.Logs;
+using System;
+
+namespace ProKnow.Test.LogTest
+{
+    [TestClass]
+    public class AuditTest
+    {
+        private static readonly string _testClassName = nameof(AuditTest);
+        private static readonly ProKnowApi _proKnow = TestSettings.ProKnow;
+
+        [ClassInitialize]
+#pragma warning disable IDE0060 // Remove unused parameter
+        public static async Task ClassInitialize(TestContext testContext)
+#pragma warning restore IDE0060 // Remove unused parameter
+        {
+            // Cleanup from previous test stoppage or failure, if necessary
+            await ClassCleanup();
+        }
+
+        [ClassCleanup]
+        public static async Task ClassCleanup()
+        {
+            // Delete test workspaces
+            await TestHelper.DeleteWorkspacesAsync(_testClassName);
+
+        }
+
+        [TestMethod]
+        public async Task NextBeforeQueryTest()
+        {
+            try
+            {
+                var receivedAuditLogItem = await _proKnow.Audit.Next();
+            }
+            catch (Exception e)
+            {
+                Assert.AreEqual(e.Message, "Must call Query first");
+            }
+        }
+
+        [TestMethod]
+        public async Task QueryTest()
```

Comment:
Add another test to test that all query parameters are working and you're not getting 422 errors